src/plugins/NeedsDocsLabel/needs_docs_label.ts

```diff
@@ -0,0 +1,50 @@
+import { PRContext } from "../../types";
+import { Application } from "probot";
+import { ORG_ESPHOME, REPO_CORE, REPO_DOCS } from "../../const";
+import { filterEventByRepo } from "../../util/filter_event_repo";
+import { getIssueFromPayload } from "../../util/issue";
+import { extractIssuesOrPullRequestMarkdownLinks, extractPullRequestURLLinks } from "../../util/text_parser";
+
+const NAME = "NeedsDocsLabel";
+
+export const initNeedsDocsLabel = (app: Application) => {
+  app.on(
+    [
+      "pull_request.labeled",
+      "pull_request.unlabeled",
+      "pull_request.edited",
+    ],
+    filterEventByRepo(NAME, ["github-test"], runNeedsDocsLabel)
```

Comment:
Hmm, on second look it appears this never triggers because `github-test` is not one of our repos.

Can you confirm this should be `REPO_CORE` ?

Reply:
Haha yes, I noticed this yesterday as well, didn't get around to submitting a PR for it yet. `github-test` was indeed the repository I used to test this and it should be `REPO_CORE`.

Reply:
No problem :) Fixed here: https://github.com/esphome/probot-esphome/pull/3